README.md

```diff
@@ -9,37 +9,40 @@ This bundle requires the following spack-stack modules be loaded (all except `fm
+  7) zstd/1.5.2                28) g2tmpl/1.10.2            49) fiat/1.1.0         70) py-setuptools/59.4.0         91) py-ply/3.11
+  8) tar/1.34                  29) sp/2.3.3                 50) ectrans/1.2.0      71) py-numpy/1.22.3              92) py-pythran/0.12.2
+  9) gettext/0.21.1            30) ip/3.3.3                 51) atlas/0.33.0       72) py-eccodes/1.4.2             93) py-scipy/1.9.3
+ 10) libxcrypt/4.4.33          31) cmake/3.23.1             52) gsibec/1.1.2       73) py-f90nml/1.4.3              94) py-xarray/2022.3.0
+ 11) sqlite/3.40.1             32) git/2.36.0               53) gsl-lite/0.37.0    74) py-h5py/3.7.0                95) jedi-base-env/1.0.0
+ 12) python/3.10.8             33) libbacktrace/2020-02-19  54) hdf/4.2.15         75) py-cftime/1.0.3.4            96) gftl/1.8.3
+ 13) stack-python/3.10.8       34) nccmp/1.9.0.1            55) jedi-cmake/1.4.0   76) py-netcdf4/1.5.3             97) gftl-shared/1.5.0
+ 14) bacio/2.4.1               35) py-pip/23.0              56) libxt/1.1.5        77) py-bottleneck/1.3.5          98) yafyaml/0.5.1
+ 15) curl/8.0.1                36) wget/1.21.3              57) libxmu/1.1.2       78) py-packaging/23.0            99) mapl/2.35.2-esmf-8.4.2
+ 16) pkg-config/0.29.2         37) base-env/1.0.0           58) libxpm/3.5.12      79) py-numexpr/2.8.3            100) w3nco/2.4.1
+ 17) hdf5/1.14.0               38) boost/1.78.0             59) libxaw/1.0.13      80) py-six/1.16.0               101) nemsio/2.5.2
+ 18) netcdf-c/4.9.2            39) bufr/12.0.0              60) udunits/2.2.28     81) py-python-dateutil/2.8.2    102) sigio/2.3.2
+ 19) netcdf-fortran/4.6.0      40) git-lfs/3.1.4            61) ncview/2.1.8       82) py-pytz/2022.2.1            103) w3emc/2.9.2
+ 20) parallel-netcdf/1.12.2    41) ecbuild/3.7.2            62) netcdf-cxx4/4.3.1  83) py-pandas/1.4.0             104) jedi-ufs-env/unified-dev
+ 21) parallelio/2.5.9          42) openjpeg/2.3.1           63) json/3.10.5        84) py-pybind11/2.8.1           105) fms@2023.01
+```
+On a pre-configured platform that has spack-stack-1.4.1 (or 1.4.0) installed, load the spack-stack modules as described in https://spack-stack.readthedocs.io/en/1.4.1 (https://spack-stack.readthedocs.io/en/1.4.0), followed by `module load jedi-ufs-env`. On a configurable (user) platform, follow the instructions in https://spack-stack.readthedocs.io/en/1.4.1 to build the `skylab-dev` or `unified-dev` environment, then run (use the appropriate path, compiler/mpi/python versions for your system):
+```
+module use /Users/heinzell/prod/spack-stack-v1/envs/unified-dev/install/modulefiles/Core
 module load stack-apple-clang/13.1.6
 module load stack-openmpi/4.1.5
 module load stack-python/3.10.8
-module load jedi-ufs-env/skylab-dev
+module load jedi-ufs-env/unified-dev
+```
+Further, for testing of the newest `fms@2023.02-beta1` tag, verify if the platform you are on has this version available and if yes, run
```

Comment:
Does this include the change that soca needed?

Reply:
Yes!

Reply:
🎉  you rock!
we'll test with fv3-jedi too, unless @travissluka you already tested with both soca and fv3-jedi?

Reply:
Well, the tag of fms contains the changes that SOCA needs, but the ufs-weather-model, fv3-jedi-linearmodel and fv3-jedi don't work with that tag ye. See https://github.com/ufs-community/ccpp-physics/issues/90 and https://github.com/JCSDA-internal/fv3-jedi-linearmodel/issues/18 for examples. Also, see https://github.com/NOAA-GFDL/FMS/issues/1284 for bugs in the build system of that tag.

Reply:
Got it, so testing https://github.com/JCSDA-internal/fv3-bundle/pull/103 (with updates to jcsda repos) isn't expected to work?

Reply:
At least at the moment, not. Let's see how far we get this week.